src/hooks/use-custom-field-value.jsx

```diff
@@ -0,0 +1,10 @@
+import { useImperativeHandle } from 'react';
+
+export default function useCustomFieldValue(id, ref, getValue) {
```

Comment:
The `useCustomFieldValue` name sounds like a `useState` derivative.

A better name for this module is: `useCustomFieldHandle` since it is creating an imperative handle for a custom field and we want to centralize most of the logic into this module.